docs/changelog/98514.yaml

```diff
@@ -0,0 +1,34 @@
+pr: 98514
+summary: Cross-cluster search improvements - ability to exclude a cluster and cluster
+  details added to CCS response
```

Comment:
I think that your PR should be modifying the existing changelogs for previous PRs, and not adding a new changelog for itself. With that, I think the description should also be split up in two: exclude a cluster and cluster details. In short, this should be a docs PR only, without any changelog modification.